aspnet/tutorials/first-mvc-app/adding-model.rst

```diff
@@ -34,72 +31,64 @@ In the **Add Scaffold** dialog, tap **MVC Controller with views, using Entity Fr
 
 .. image:: adding-model/_static/add_scaffold2.png
 
-- Complete the **Add Controller** dialog
+- Complete the **Add Controller** dialog 
 
-  - **Model class:** *Movie(MvcMovie.Models)*
-  - **Data context class:** *ApplicationDbContext(MvcMovie.Models)*
-  - **Controller name:** Keep the default MoviesController
-  - **Views:**: Keep the default of each option checked
-  - **Controller name:** Keep the default *MoviesController*
-  - Tap **Add**
+	- **Model class:** *Movie(MvcMovie.Models)*
+	- **Data context class:** *ApplicationDbContext(MvcMovie.Models)*
+	- **Views:**: Keep the default of each option checked
+	- **Controller name:** Keep the default *MoviesController*
+	- Tap **Add**
 
 .. image:: adding-model/_static/add_controller2.png
 
 The Visual Studio scaffolding engine creates the following:
 
-- A movies controller (MoviesController.cs)
-- Create, Delete, Details, Edit and Index Razor view files
-- Migrations classes
+- A movies controller (*Controllers/MoviesController.cs*)
+- Create, Delete, Details, Edit and Index Razor view files (*Views/Movies*)
+
+Visual Studio automatically created the `CRUD <https://en.wikipedia.org/wiki/Create,_read,_update_and_delete>`__ (create, read, update, and delete) action methods and views for you (the automatic creation of CRUD action methods and views is known as *scaffolding*). You'll soon have a fully functional web application that lets you create, list, edit, and delete movie entries.
+
+Run the app and click on the **Mvc Movie** link. You'll get the following error:
+
+.. code-block:: html
 
-  - The ``CreateIdentitySchema`` class creates the :doc:`membership database </security/authentication/identity>` tables. The Identity database stores user login information that is needed for authentication. We won't cover authentication in this tutorial, for that you can follow `Additional resources`_ at the end of this tutorial.
-  - The ``ApplicationDbContextModelSnapshot`` class creates the EF entities used to access the Identity database. We'll talk more about EF entities later in the tutorial.
+  A database operation failed while processing the request.
```

Comment:
@danroth27 I used text so for search-ability, but the image looks so much better I went back to the image.